besu/src/main/java/org/hyperledger/besu/controller/IbftBesuControllerBuilder.java

```diff
@@ -208,7 +204,6 @@ protected MiningCoordinator createMiningCoordinator(
 
   @Override
   protected PluginServiceFactory createAdditionalPluginServices(final Blockchain blockchain) {
-    final NodeKey nodeKey = new BouncyCastleNodeKey(nodeKeys);
     return new IbftQueryPluginServiceFactory(blockchain, nodeKey);
```

Comment:
same as before, getNodeKey() instead of nodeKey??

Reply:
as above - trying to leave things untouched.